cmd/sc/sc.go

```diff
@@ -0,0 +1,57 @@
+package main
+
+import (
+	"log"
+
+	"github.com/urfave/cli"
+
+	"github.com/evrynet-official/evrynet-tools/staking"
+)
+
+func vote(ctx *cli.Context) {
+	stakingClient, err := staking.NewNewStakingFromFlags(ctx)
+	if err != nil {
+		log.Printf("can not initialize a staking contract client %s", err.Error())
+		return
+	}
+	_, err = stakingClient.Vote()
+	if err != nil {
+		log.Printf("un-vote for candidate %s got error: %w", stakingClient.Candidate.Hex(), err)
```

Comment:
What is `%w` mean?

Reply:
i've reference from ```In Go 1.13, the fmt.Errorf function supports a new %w verb. When this verb is present, the error returned by fmt.Errorf will have an Unwrap method returning the argument of %w, which must be an error. In all other ways, %w is identical to %v."```

Reply:
But this project is using go 1.12, you can check it on go.mod. So please change it ^^